OSDOCS-5637-batch1: Documented the batch #1 of RN bug fixes (#57943)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

[id="ocp-4-13-management-console-bug-fixes"]
==== Management Console

> * Previously, a terminated container would render as a meaningless string in the internationalization code results. For example, the `{{label}}` and {{exitCode}}' codes would display in the results with no human-readable string output, such as `Terminated with exit code 13` for each terminated container. The {product-title} {product-version} fixes this issue with an update to the internationalization code that now ensures a human-readable string exists in the internationalization code results for a container that was terminated. (link:https://issues.redhat.com/browse/OCPBUGS-4206[*OCPBUGS-4206])

Author: Sorry, @opayne1 . ocpbugs-4206 also.

> * Previously, a terminated container would render as a meaningless string in the internationalization code results. For example, the `{{label}}` and {{exitCode}}' codes would display in the results with no human-readable string output, such as `Terminated with exit code 13` for each terminated container. The {product-title} {product-version} fixes this issue with an update to the internationalization code that now ensures a human-readable string exists in the internationalization code results for a container that was terminated. (link:https://issues.redhat.com/browse/OCPBUGS-4206[*OCPBUGS-4206])

Reviewer: Feel free to keep yours, but you could cut it down to follow some of the minimalism guidelines.

```suggestion
* Previously, a terminated container would render `{{label}}` and {{exitCode}}' codes  for each terminated container. With this update,  the internationalization code is fixed to render a readable output message.(link:https://issues.redhat.com/browse/OCPBUGS-4206[*OCPBUGS-4206])
```

* Previously, if a user with regular account privileges attempted to issue the `oc delete csv` command when uninstalling the Operator, the Operator's subscription becomes stuck. This issue occurs because the user does not have access to remove subscription resources. After this action, an administrator cannot reinstall the uninstalled Operator, because a conflict exists with the stuck subscription. {product-title} {product-version} fixes this issue by providing a detailed error message that displays when an administrator attempts to reinstall the uninstalled Operator. (link:https://issues.redhat.com/browse/OCPBUGS-3822[*OCPBUGS-3822])

> * In the {product-title} 4.12 release, the node terminal tab would display an error because of missing annotations that were caused by pod security measures. Without the required annotations, the node debug pod cannot start. The {product-title} {product-version} adds these annotations, so that the node debug pod can start and the node terminal tab loads without any errors. (link:https://issues.redhat.com/browse/OCPBUGS-4252[*OCPBUGS-4252]) 

Author: Hi @opayne1 . I think you might be the writer for management console components?

OCPBUGS-4252 had CCFR text, but I went ahead to write the note. I know not to do this from now on. Would you be OK to review my note or provide your own?

Reviewer: Just a note: instead of using {product-title} {product-version}, it would be better to use "with this update" or something similar in case this bug is replicated on another version, the copied text from JIRA would be more accurate.
